[expo-payments-stripe] Upgrade Stripe dependency to fix Xcode 11.4 compat (#8594)
expo/expo: packages/expo-payments-stripe/CHANGELOG.md

### 🎉 New features

> - Added the ability to customize version of `Stripe` pod installed by reading the `$StripeVersion` variable when installing the pod. ([#8594](https://github.com/expo/expo/pull/8594) by [@sjchmiela](https://github.com/sjchmiela))
> 
> ### 🐛 Bug fixes
> 
> - Upgraded `Stripe` pod on iOS to fix compatibility with Xcode 11.4. ([#8594](https://github.com/expo/expo/pull/8594) by [@sjchmiela](https://github.com/sjchmiela))
> 

Author: ```suggestion

### 🐛 Bug fixes

- Upgraded `Stripe` pod on iOS to fix compatibility with Xcode 11.4. Now you can also customize the version of `Stripe` pod installed by setting `$StripeVersion` variable in your `Podfile`. ([#8594](https://github.com/expo/expo/pull/8594) by [@sjchmiela](https://github.com/sjchmiela))

```
